src/main/java/com/android/ex/chips/RecipientEditTextView.java

```diff
@@ -3163,6 +3163,16 @@ public BaseRecipientAdapter getAdapter() {
      * unfinished text at the end.
      */
     public void appendRecipientEntry(final RecipientEntry entry) {
+        if(getMeasuredWidth() == 0){
+            post(new Runnable() {
+                @Override
+                public void run() {
+                    appendRecipientEntry(entry);
+                }
+            });
+            return;
+        }
```

Comment:
Do you know why this is needed? Doesn't seem like it's related to the identity id